src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,30 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo
+          user={post.user}
+        />
```

Comment:
user can be null, let's handle it here
```suggestion
        {post.user && (<UserInfo
          user={post.user}
        />)}
```